### ISSUE TITLE
using CLI from master shows error for Java func

### ISSUE THREAD

**Issue by trisberg** [projectriff/riff#386]
Building CLI from master and creating a Java function I get this for the function pod:

```
  Warning  InspectFailed          6s (x9 over 1m)  kubelet, minikube  Failed to apply default image tag "/upper:0.0.1": couldn't parse image reference "/upper:0.0.1": invalid reference format
  Warning  Failed                 6s (x9 over 1m)  kubelet, minikube  Error: InvalidImageName
```

so it looks like we are missing the user-account part

**Comment by scothis** [comment]
Seeing something similar for other functions in FATS.

**Comment by trisberg** [comment]
Yes, looks like node has the same issue, haven't tried other languages

**Comment by trisberg** [comment]
A workaround is to provide RIFF_USERACCOUNT env var

**Comment by trisberg** [comment]
Looks like this is fixed via commit d39f946efcc43aa53b6a1e1f36c7293dc0451a0b

**Comment by dturanski** [comment]
I'm re opening this.  

---
riff create python
...
Initializing /Users/dturanski/go/src/github.com/projectriff/riff/samples/python/concat/Dockerfile
Building image ...
[STDERR] invalid argument "/concat:0.0.1" for "-t, --tag" flag: invalid reference format
[STDERR] See 'docker build --help'.
---
